packages/compile-solidity/compilerSupplier/index.js

```diff
@@ -4,10 +4,11 @@ const semver = require("semver");
 
 const { Docker, Local, Native, VersionRange } = require("./loadingStrategies");
 
+const defaultSolcVersion = "0.5.16";
```

Comment:
Do we want to make this name all caps to denote that it is a constant?

Reply:
Hm, I was just leaving it as-is.  I think @gnidan has said we should be moving away from all-caps anyway?

Reply:
Yeah, I'm leaning towards "name constants like normal" these days